src/main/java/org/jongo/RxJongo.java

```diff
@@ -0,0 +1,46 @@
+package org.jongo;
```

Comment:
Note that there is another repository : https://github.com/bguerout/jongo-extras with other implementations/recipes.
It should be nice to add a RxJongo folder.
What do you think about that ?